components/dashboard/src/service/service.tsx

```diff
@@ -110,7 +110,7 @@ export class IDEFrontendService implements IDEFrontendDashboardService.IServer {
             // send last heartbeat (wasClosed: true)
             const data = { sessionId: this.sessionId };
             const blob = new Blob([JSON.stringify(data)], { type: "application/json" });
-            const gitpodHostUrl = new GitpodHostUrl(new URL(window.location.toString()));
```

Comment:
cc @mustard-mh should be alright?

Reply:
It's not working. I thought we should keep `spec` file and add our own test cases
<img width="1324" alt="image" src="https://user-images.githubusercontent.com/20944364/234303751-f85ecd9d-8198-46aa-88de-b2ba8326c8ed.png">

Reply:
I see, the `with` is not statically typed, so compiler does not complain. 👍

Reply:
Yep, we want to get gitpod host here only